src/main/java/uk/gov/laa/pfla/auth/service/services/ReportTrackingTableService.java

```diff
@@ -46,11 +46,8 @@ public synchronized void updateReportTrackingTable(int requestedId, LocalDateTim
                 .reportGeneratedBy(user.getGivenName() + " " + user.getSurname())
                 .build();
 
-        if(log.isDebugEnabled()){
-            log.debug("reportTrackingTableModel to string: {} ", reportTrackingTableModel);
-        }
-
-        //Create a trackingtable row in the table
+        //Create a new trackingtable row
+        log.info("Updating reportTrackingTable with details of the report request from user");
```

Comment:
From a security point of view Is it required to mention table name in the info logs? can we remove table name? mention something like "updating report tracking details" in you want to keep this in info log